packages/graphiql-react/src/icons/index.ts

```diff
@@ -0,0 +1,77 @@
+import { ComponentProps, FC } from 'react';
```

Comment:
hmm... I'm used to doing something like this in a vite plugin in my proprietary work, is there a way we can configure the plugin to do this, or copy the plugin source and tweak it as such?

Reply:
svgr plugin incorrectly shows the type `string` for imported svg files, even though we set up `exportAsDefault: true` in vite config, if you know how to fix it feel free